### PR TITLE
(feat) O3-2580: Use the new snackbar in other patient chart apps(part 3)

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
- Use the new snackbar in all places where `Toast` and `Notification` were used in patient chart `immunizations`, `notes` , `programs`, `medications`, `labs` and `vitals` app.

## Screenshots
- Sample screenshot for the snackbar
Screenshot 2023-11-20 at 02 08 27

## Related Issue
- https://issues.openmrs.org/browse/O3-2580

### CODE REVIEW

File: packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx [hunk 1]
```diff
@@ -79,17 +79,18 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
...
           });
         },
         (err) => {
-          showNotification({
+          showSnackbar({
             title: t('errorSaving', 'Error saving vaccination'),
             kind: 'error',
-            critical: true,
-            description: err?.message,
+            isLowContrast: false,
```
Comment (vasharma05) [inline packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx:92]: If we have low contrast for success and high for error, isn't it better to make it low contrast in the notification component itself?
```
lowContrast : props.lowContrast || props.kind !== 'error'
```
```

File: packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx [hunk 2]
```diff
@@ -226,21 +225,21 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({ closeWorkspace, patie
           mutateVisitNotes();
           closeWorkspace();
 
-          showToast({
-            critical: true,
-            description: t('visitNoteNowVisible', 'It is now visible on the Encounters page'),
+          showSnackbar({
+            isLowContrast: true,
```
Comment (vasharma05) [inline packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx:229]: We have to pass this again and again.

File: packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx (hunk 1 above)
Comment (hadijahkyampeire) [reply packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx:92]: I will try it and see if it works generically.

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @hadijahkyampeire!

**Pull request merged**